Frameworks/UIKit/StarboardXaml/ApplicationMain.mm

```diff
@@ -179,6 +179,6 @@ int ApplicationMainStart(const char* principalName,
     return 0;
 }
 
-void SetTemporaryFolder(const char* folder) {
-    NSSetTemporaryDirectory([NSString stringWithCString:folder]);
+void SetTemporaryFolder(const wchar_t* folder) {
+    NSSetTemporaryDirectory([NSString stringWithCharacters:(const unichar*)folder length:wcslen(folder)]);
```

Comment:
Yes it will be copied there anyway. It would still save a copy to not have a useless extra layer here to set it ... 

---
In reply to: [99445013](https://github.com/Microsoft/WinObjC/pull/1916#discussion_r99445013) [](ancestors = 99445013)